modules/EventTracker.js

```diff
@@ -0,0 +1,187 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', 'backbone', './ServiceCell'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('backbone'), require('./ServiceCell'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.EventTracker = factory(root._, root.Backbone, root.Torso.ServiceCell);
+  }
+}(this, function(_, Backbone, ServiceCell) {
+  'use strict';
+
+  var EventTracker = ServiceCell.extend({ 
```

Comment:
@mandragorn, I'm having a hard time tracking what parts of my comments you are referring to. But, I think that we are in agreement that we probably should trigger two events, one on the instance (model, view, or router) and one on the Events. Listening to Events will let us attach system-level logic while listening to the instance is more granular. For instance triggering, would it make sense to have automatic callbacks, or will I have to add this.on('my-pre-event', this.preEvent) and this.on('my-post-event', this.postEvent) on each view?